crates/prover-executor/src/lib.rs

```diff
@@ -27,16 +27,14 @@ mod error;
 
 #[derive(Clone)]
 pub struct Executor {
+    vkey: SP1VerifyingKey,
```

Comment:
```suggestion
    vkey: Arc<SP1VerifyingKey>,
```

Do you think we could `arc` this?